woocommerce/woocommerce-android
# [Woo POS] Start card reader payment flow

Closes: #11609


### Description

Implements starting of the payment flow and returning of the result of it

### Testing instructions

**The most important to make sure there is no regression in the current IPP flow**

To the the POS flow click on the button on the root screen

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/07785a10-4c57-404b-9580-3e6b8df5a4e6
